rush.json

```diff
@@ -686,8 +686,7 @@
       "packageName": "@rushstack/heft-jest-plugin",
       "projectFolder": "heft-plugins/heft-jest-plugin",
       "reviewCategory": "libraries",
-      "shouldPublish": true,
-      "cyclicDependencyProjects": ["@rushstack/heft", "@rushstack/heft-node-rig"]
```

Comment:
Can this project just take a cyclic version of itself and manually install itself in its `config/heft.json`?

Reply:
It can't unfortunately. This project itself has a peerDependency on Heft, which (if a cyclic dependency is used) would cause the versions being used to go out-of-spec on minor version bumps (for `0.x.x` versions of Heft) or major version bumps (for `y.x.x` versions of Heft, where `y >= 1`.

Instead, I've added the `RunScriptPlugin` and added two bootstrapping scripts that run the plugin code directly after it is built. This allows us to use our own built assets to test our code.